vali_objects/utils/position_manager.py

```diff
@@ -34,7 +35,11 @@ def close_open_positions_for_miner(self, hotkey):
                     self.save_miner_position_to_disk(open_position)
 
 
-    def get_return_per_closed_position(self, positions: List[Position]) -> List[float]:
+    def get_return_per_closed_position(
```

Comment:
Seems to me that the original calculation of using position.return_at_close no longer exists. This would break the existing logic in MDDChecker